openedx/features/idea/models.py

```diff
@@ -2,5 +2,77 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from django.contrib.auth.models import User
+from django.core.validators import FileExtensionValidator
+from django_countries.fields import CountryField
 
-# Create your models here.
+from functools import partial
+
+from lms.djangoapps.onboarding.models import Organization
+from .helpers import upload_to_path
+
+
+class Location(models.Model):
+    country = CountryField()
+    city = models.CharField(max_length=255)
+
+    @property
+    def location(self):
```

Comment:
How will it be used? because it is an abstract Model, we will never have Location object how will access this property?

Reply:
Our main model `Idea` has inherited this abstract model, this property will be accessible though `Idea`.